### PR TITLE
Basic firebaseopensource config

### PR DESCRIPTION
**Pull request by samtstern**

### CODE REVIEW

**Comment by samtstern** [comment]
99% sure the Travis failures are unrelated since this PR only adds a JSON file in a previously unknown directory.

File: .opensource/project.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        "Admin"
+    ],
+    "content": "README.md",
+    "pages": [],
+    "related": [
+        "firebase/firebase-admin-java",
+        "firebase/firebase-admin-go",
+        "firebase/firebase-admin-python"
+    ]
+}
```
Comment (hiranya911) [inline .opensource/project.json]: Nit: add new line at eof

**Comment by hiranya911** [comment]
Build break is due to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/27943, a breaking change which they released as 1.7.1 :(

**Comment by hiranya911** [comment]
Build will be back to normal once #340 is merged.

**Pull request merged**